HomeKitBridge/characteristics/pm25Density.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
Hatte ich erst vor, aber in der Doku von Apple wurde es so geschrieben. Wollte die Konstanten evtl. auch noch anpassen.
Welche Schreibweise ziehst du vor?